recipe/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
 
 build:
   skip: true  # [win]
-  number: 0
+  number: 1000
```

Comment:
Missing compiler declaration. We need to run update-cb3 here.

Reply:
I may be wrong. I could not find a compiled extension here. Let me try to remove the `{{ compiler('c') }}` and re-render.